src/ale_interface.cpp

```diff
@@ -254,3 +254,7 @@ void ALEInterface::restoreState(const ALEState& state) {
 void ALEInterface::saveScreenPNG(const string& filename) {
   environment->saveScreenPNG(filename);
 }
+
+ScreenExporter *ALEInterface::createScreenExporter(const std::string &filename) const {
```

Comment:
Following on my previous comments I'd suggest hiding the ScreenExporter behind StellaEnvironment (and removing the createScreenExporter method). If the user wants to record the whole trajectory, (e.g. the saveNext method of ScreenExporter) they can do it by giving ALE the -record_screen_dir flag and the environment will take care of it. If the want to save individual screens they can use the saveScreenPNG method above.